src/index.html

```diff
@@ -6,9 +6,28 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img class="logo"
+        src="images/logo.png"
+        alt="Moyo logo">
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a class="nav-link is-active">Apple</a></li>
+          <li class="nav-item"><a class="nav-link">Samsung</a></li>
```

Comment:
```suggestion
          <li class="nav-item">
            <a class="nav-link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav-item">
            <a class="nav-link">
              Samsung
            </a>
          </li>
```
Such formatting will help to improve readability